lambdas/drivingpermitcheck/src/test/java/uk/gov/di/ipv/cri/drivingpermit/api/service/IdentityVerificationServiceTest.java

```diff
@@ -149,48 +128,46 @@ void verifyIdentityShouldReturnErrorWhenThirdPartyCallFails()
         verify(mockEventProbe).counterMetric(DCS_CHECK_REQUEST_FAILED);
     }
 
-    @Test
-    void verifyUseLegacyParameterRoutesUsersToDvaWhenFalse()
-            throws IOException, InterruptedException, CertificateException, ParseException,
-                    JOSEException, OAuthHttpResponseExceptionWithErrorBody {
-        try (MockedStatic<LogManager> mockedLogManager = mockStatic(LogManager.class)) {
-            Logger mockedStaticLogger = mock(Logger.class);
-            mockedLogManager.when(LogManager::getLogger).thenReturn(mockedStaticLogger);
-
-            when(configurationService.getUseLegacy()).thenReturn(false);
-            this.identityVerificationService =
-                    new IdentityVerificationService(
-                            mockThirdPartyGateway,
-                            formDataValidator,
-                            mockContraindicationMapper,
-                            mockAuditService,
-                            configurationService,
-                            objectMapper,
-                            mockEventProbe);
-            DrivingPermitForm drivingPermitForm = DrivingPermitFormTestDataGenerator.generate();
-            DocumentCheckResult testFraudCheckResult = new DocumentCheckResult();
-            testFraudCheckResult.setExecutedSuccessfully(true);
-            String[] thirdPartyFraudCodes = new String[] {"sample-code"};
-            String[] mappedFraudCodes = new String[] {"mapped-code"};
-            testFraudCheckResult.setValid(true);
-            when(formDataValidator.validate(drivingPermitForm))
-                    .thenReturn(ValidationResult.createValidResult());
-            when(mockThirdPartyGateway.performDocumentCheck(drivingPermitForm))
-                    .thenReturn(testFraudCheckResult);
-
-            DocumentCheckVerificationResult result =
-                    this.identityVerificationService.verifyIdentity(drivingPermitForm);
-
-            assertNotNull(result);
-            verify(formDataValidator).validate(drivingPermitForm);
-            verify(mockEventProbe).counterMetric(FORM_DATA_VALIDATION_PASS);
-            /*
-            TODO: This below line will need to be updated to new performDvaDocumentCheck method once
-            created, also remove the verify logline and make logger in class static again
-             */
-
-            verify(mockThirdPartyGateway).performDocumentCheck(drivingPermitForm);
-            verify(mockedStaticLogger).info("Performing document check (DVA direct)");
-        }
-    }
+    //    @Test
```

Comment:
Why is this test commented out?

Reply:
It was originally confirming a log line. But no longer applicable as the switch is done in the handler.